package.json

```diff
@@ -9,7 +9,7 @@
     "start": "node dist/index.js",
     "test": "nyc --reporter=html --reporter=text mocha ./test/*.js --timeout 80000 --exit --require @babel/register --require regenerator-runtime",
     "devtest": "NODE_ENV=test & yarn destroy & yarn run seed & yarn run test",
-    "test:reset": "NODE_ENV=test npm run migrate:reset && NODE_ENV=test nyc --reporter=html --reporter=text mocha ./test/*.js --timeout 80000 --exit --require @babel/register --require regenerator-runtime",
+    "test:reset": "NODE_ENV=test & npm run migrate:reset && NODE_ENV=test nyc --reporter=html --reporter=text mocha ./test/*.js --timeout 80000 --exit --require @babel/register --require regenerator-runtime",
```

Comment:
`NODE_ENV=test` is already present on Travis, is there of including it here still ?